x-pack/plugins/observability_solution/observability_ai_assistant_app/scripts/evaluation/kibana_client.ts

```diff
@@ -357,7 +385,11 @@ export class KibanaClient {
         } else if (args.length === 2 && !isMessageList(args[1])) {
           messagesArg = args[0];
           options = args[1];
-        } else if (args.length === 2 && typeof args[0] === 'string' && isMessageList(args[1])) {
+        } else if (
+          args.length === 2 &&
+          (typeof args[0] === 'string' || typeof args[0] === 'undefined') &&
+          isMessageList(args[1])
+        ) {
```

Comment:
This reminds me of untyped code.

Reply:
yes but it's actually typed without using any type casts 😄 I know it doesn't look great but I want the function to be as simple as possible but still allow for more advanced use cases, as we expect non-Kibana folks to contribute here.